_posts/current/blog/2020-07-02-linaro-engineering-highlights-june-2020.md

```diff
@@ -0,0 +1,159 @@
+
+#### **Working on VirtIO**
+
+By Alex Bennée {% include image.html path="/assets/images/content/virtio.png" alt="Virtio diagram" %}
+
+Linaro recently published an [overview of VirtIO work](https://www.linaro.org/blog/virtio-work/) written by Alex Bennée. Alex reviews the early history of virtualization, discusses the different types of virtualization and what the active collaborative work is. Standardization and upstreaming are key parts of this effort.
+
+#### **June LITE Sprint**
+
+By Kumar Gala, LITE Technical Lead and Vicky Janicki, LITE Engineering Director
+
+{% include image.html path="/assets/images/content/lite.jpg" class="small-inline left" alt="Lite icon" %}Utilizing conferencing capabilities, the LITE team met the week of June 15th to focus on CI/Testing and automation. The goals included updating to the latest version of tools including LAVA, running a basic series of tests on their development systems and discussing and planning the next steps. The LITE team has been working closely with the LAVA and Lab teams over the past year to solve the problem of consistently running tests for MCUs (Cortex-M devices). Using the LAVA framework, the LAVA and Lab teams invented a Docker solution to cope with the many boards and environments that the LITE team is using. The vision is to have a framework or environment that is reliable and consistent while providing easy paths to test new software projects and tools in the future. 
+
+Under the CI umbrella, David Brown (Linaro) worked to update MCUboot to use Github’s CI in place of Travis CI and then add to the Zephyr integration testing. Ed Mooring (Xilinx) worked on extracting the toolflow from Xilinx internal build system for building a test environment for OpenAMP on Cortex-R5 and QEMU. Future plans include completing the MCUboot CI, successfully building and running OpenAMP using tools based outside Xilinx and starting a TF-M CI for LITE. 
+
+{% include image.html path="/assets/images/content/lite-zephyr-upstream.png" "small-inline left" alt="Zephyr build information" %}The team also took  a deep look into how they are using LAVA and the reporting application SQUAD. Dan Rue and Daniel Diaz from the LKFT team and Remi Duraffort from the LAVA project reviewed how LKFT uses LAVA and SQUAD and walked through some of the design decisions. They then met with the Mbed Green Tea team from Arm, the LAVA team and the Zephyr Testing Working Group Team. The result of the meeting was to start prototyping the LAVA Docker Test Action as a means to encapsulate outside testing tools such as Zephyr sanitycheck or Mbed’s GreenTea. A key requirement is that the testing flow and tools should be the same for a developer at their desk as for the LAB CI/Test environment. 
```

Comment:
You're missing a `class=` before 
 "small-inline left"

Reply:
Thanks!